### PR TITLE
Don't need to copy bootstrap folder.

### PR DESCRIPTION
**Pull request by kesali**

### CODE REVIEW

**Comment by jaesung2061** [comment]
Are you on Windows currently?

**Comment by kesali** [comment]
yes i am on windows

**Comment by jaesung2061** [comment]
I'm guessing this is because of the new Window Bash? Is this now standard on all Windows devices? If not, I need to keep this in the docs.

**Comment by kesali** [comment]
I use powershell nothing really special here. I also tested on standart command prompt everything works as expected. 
Thanks

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,4 @@ Homestead.json
 Homestead.yaml
 npm-debug.log
 .env
+composer.lock
```
Comment (jaesung2061) [inline .gitignore]: Need to remove this, I will merge after.

**Comment by kesali** [comment]
Done

**Pull request merged**